src/app/src/features/eu4/features/country-details/CountryDetailsDescriptions.tsx

```diff
@@ -71,14 +71,17 @@ export const CountryDetailsDescriptions = ({
           <tbody>{ideaElem}</tbody>
         </table>
       </Descriptions.Item>
-      <Descriptions.Item label={<InheritanceLabel details={details} />}>
+      <Descriptions.Item label={<InheritanceLabel details={details} />} 
+        span={3}>
         <div className="no-break flex flex-col">
           {!isJuniorParter && (
             <div>{`Window: [${details.inheritance.start_t1_year} - ${details.inheritance.end_t1_year}]`}</div>
           )}
           <div>{`Inheritance Value: ${details.inheritance.inheritance_value}`}</div>
         </div>
       </Descriptions.Item>
+
+      <Descriptions.Item label="Development">{`${details.raw_development} (${details.development} counting autonomy)`}</Descriptions.Item>
```

Comment:
Awesome. can you use `formatFloat` and/or `formatInt` functions to format the numbers so that the decimal and thousandth place separator are localized?